src/components/GetStarted.tsx

```diff
@@ -229,99 +238,237 @@ const GetStarted = ({
+								<ExternalLink
+									label="Rosetta@home system requirements"
+									href="https://boinc.berkeley.edu/wiki/List_of_projects_by_system_requirements"
+								/>
+								, you may also want to check out their{' '}
+								<ExternalLink
+									label="rules and policies"
+									href="http://boinc.bakerlab.org/rosetta/info.php"
+								/>
+								.
+							</Txt.p>
+						</Tab>
 					</Tabs>
 
 					<Heading.h3 bold my={3}>
 						Let's begin
 					</Heading.h3>
-					<Step index={1}>
-						<Txt fontSize={2} bold>
-							Download and Install{' '}
-							<ExternalLink
-								href="https://balena.io/etcher"
-								label="balenaEtcher"
-							/>
-						</Txt>
-						<DownloadEtcher />
-					</Step>
-					<Step index={2}>
-						<Txt fontSize={2} bold>
-							Download the ready-made Operating System below.
-						</Txt>
-						<DownloadImage
-							sdk={sdk}
-							selectedApp={selectedApp}
-							selectedDeviceType={selectedDeviceType}
-						/>
-					</Step>
-					<Step index={3}>
-						<Txt fontSize={2} bold>
-							Launch balenaEtcher
-						</Txt>
-						<Txt my={3}>
-							Choose the file you just downloaded, select your SD card and click
-							"Flash".
-						</Txt>
-						<Flex
-							alignItems="center"
-							justifyContent="center"
-							maxHeight="300px"
-							maxWidth="400px"
-							pt={2}
-						>
-							<LazyImage src={flashCard} alt="Flash card with Etcher" />
-						</Flex>
-					</Step>
-					<Step index={4}>
-						<Txt>
-							Once complete,{' '}
-							<Txt.span bold>
-								place the SD Card in your {selectedDeviceType?.name}, and power
-								it on
-							</Txt.span>
-							.
-						</Txt>
-						<Flex
-							alignItems="center"
-							justifyContent="center"
-							maxHeight="300px"
-							maxWidth="400px"
-							pt={2}
-						>
-							<LazyImage src={insertCard} alt="Insert card in device" />
-						</Flex>
-					</Step>
-					<Step index={5}>
-						<Txt>
-							<Txt.span>
-								Your {selectedDeviceType?.name} will automatically join the
-								Fleet, and begin crunching data!
-							</Txt.span>
-						</Txt>
-					</Step>
-					<Step index={6}>
-						<Txt>
-							<Txt.span>
-								To view your {selectedDeviceType?.name}'s current activity,
-								visit your {selectedDeviceType?.name}’s new hostname,
-								foldforcovid.local, in a web browser like this:{' '}
-							</Txt.span>
-							<ExternalLink
-								href="http://foldforcovid.local"
-								label="foldforcovid.local"
-							/>
-							.
-						</Txt>
-						<Flex
-							alignItems="center"
-							justifyContent="center"
-							maxHeight="300px"
-							maxWidth="500px"
-							pt={2}
-						>
-							<LazyImage src={tasksImg} alt="Rosetta tasks on your device" />
-						</Flex>
-					</Step>
+					{isMainComputer ? (
+						<>
+							<Step index={1}>
+								<Txt fontSize={2} mb={2} bold>
+									Download and Install BOINC
```

Comment:
can we link to appropriate installer based on user agent as we do for etcher?

Reply:
Yes, I'll work on it next, wanted to get something out before doing that, so  it can be reviewed.

Reply:
aewsome! thanks. I agree it is not high priority